### ISSUE TITLE
[Feature request] Get PublisherOptions and SubscriberOptions from introspection types

### ISSUE THREAD

**Issue by lmiccio** [eclipse-iceoryx/iceoryx#2076]
Curently, it seems that there is no PublisherOptions or SubscriberOptions in introspection types. It would be very useful to have this information stored in the introspection types when discoverying current topics.

This could be useful when creating subscribers or publishers dynamically based on topics options.

Is there any plan to add such a feature?
Thanks.

**Comment by elBoberido** [comment]
No, there is currently no plan but it is definitely a good idea. If you have the knowledge and time a PR would be welcome :)
